fastify.d.ts

```diff
@@ -281,6 +281,7 @@ declare namespace fastify {
     method?: HTTPMethod,
     authority?: string,
     headers?: object,
+    query?: object,
```

Comment:
You can use `DefaultQuery` type for headers and query.

Reply:
There is a `DefaultHeaders`...